.dockerignore

```diff
@@ -14,9 +14,7 @@ CONTRIBUTORS
 docker-compose.*
 Dockerfile
 docs
-LICENSE
```

Comment:
`LICENSE` and `README.md` needs to be included in the Docker image, because hatch needs those files to build the package.